The-Chonky-Panda/Atelier
# Product information component skeleton

## Description
Created components for product info, name/price, styles, select quantity and size, and description. Nothing's functional yet but most of the pertinent data from the api is rendered. 

## How to test
Manually, start the server, go to localhost:3000 and see that everything renders.

## Notes

- Still need to add 'features'
- Need to add sales price

## Issue tracking
https://trello.com/c/koPgIXIj/93-create-product-information-component
